Assets/_Application/Layers/LayerTypes/GeoJsonLayers/GeoJSONPointLayer.cs

```diff
@@ -125,19 +125,28 @@ public void AddAndVisualizeFeature<T>(Feature feature, CoordinateSystem original
 
         public override void ApplyStyling()
         {
-            pointRenderer3D.Material = GetMaterialInstance();
+            // The color in the Layer Panel represents the default fill color for this layer
+            LayerData.Color = LayerData.DefaultSymbolizer?.GetFillColor() ?? LayerData.Color;
+
+            // TODO: We implement per-feature styling in a separate story; this means that for styling purposes
+            //   we consider this whole layer to be a single feature at the moment
+            var features = GetFeatures<BatchedMeshInstanceRenderer>();
+            var style = GetStyling(features.FirstOrDefault());
+            var color = style.GetFillColor() ?? Color.white;
+            
+            pointRenderer3D.Material = GetMaterialInstance(color);
```

Comment:
op het moment dat er een nieuwe material ge-assigned wordt moet de oude denk ik nog opgeruimd worden ivm memory leaks, zeker als de gebruiker continu nieuwe kleuren kan instellen

Reply:
Is aangepakt middels een herbruikbare MaterialApplicator klasse